applications/GettingStartedWith/Toast/index.js

```diff
@@ -0,0 +1,42 @@
+$(function () {
+    const types = ['error', 'info', 'success', 'warning'];
+
+    $("#show").dxButton({
+        text: "Show message",
+        onClick: function() {
+            DevExpress.ui.notify({
+                message: "You have a new message", 
+                width: 230,
+                position: {
+                    my: "bottom",
+                    at: "bottom",
+                    of: "#container"
+                }
+            }, 
+            types[Math.floor(Math.random() * 4)], 
+            500
```

Comment:
indents are not intuitive to me. Shouldn't we increase it for parameters?

Reply:
Noted